sigstore/root-signing
# snapshot-timestamp for asraa

Signed-off-by: Asra Ali <asraa@google.com>



#### Summary


#### Release Note


#### Documentation


**TO VERIFY:**
```
REPO=$(pwd)/ceremony/2022-07-12  ./scripts/verify.sh 321
```

You should see
`timestamp.json` and `snapshot.json` validly signed under staged metadata, with versions 38.

You can also manually verify the hashes.
for.e.g

```
$ sha256sum $(pwd)/ceremony/2022-07-12/staged/targets.json
730ce11e76f9c91f3814556f97f84566d95f6908b246a0013d7d66db52167117  /home/asraa/git/root-signing/ceremony/2022-07-12/staged/targets.json
$  cat $(pwd)/ceremony/2022-07-12/staged/snapshot.json | jq -r '.signed.meta."targets.json"'
{
  "hashes": {
    "sha256": "730ce11e76f9c91f3814556f97f84566d95f6908b246a0013d7d66db52167117",
    "sha512": "7c6d8beacd11aa9011a700a0509fead75fcf54d836a414f5633a2c29ee3c10415797a38aa85396953a3c6b206a052225852ce2cc85ae8b110d849855d32d0f59"
  },
  "length": 5984,
  "version": 4
}


```
